capsule.bxb

```diff
@@ -33,6 +33,14 @@ capsule {
     all
   }
   permissions {
-    library-permission (self:profile) // Note: While this permission is declared, it is not being used and the user will not be prompted, because we are fetching an 'imaginary' mocked profile from viv.self.  // This is done by passing an 'imaginary' flag to the self.GetProfile action, which we do on line 17 of models/actions/book/user-goals/CreateOrder.model.bxb: `value: self.Imaginary(true)` // Commenting out that line will cause viv.self to fetch the user's profile instead of a mocked profile, and the user will be prompted for this 'profile' permission when viv.self tries to fetch it.
+    // Note: While this permission is declared, it is not being used
+    // and the user will not be prompted, because we are fetching an
+    // 'imaginary' mocked profile from viv.self.  // This is done by
+    // passing an 'imaginary' flag to the self.GetProfile action, which
+    // we do on line 17 of models/actions/book/user-goals/CreateOrder.model.bxb: `value: self.Imaginary(true)`
+    // Commenting out that line will cause viv.self to fetch the user's
+    // profile instead of a mocked profile, and the user will be prompted
+    // for this 'profile' permission when viv.self tries to fetch it.
+    library-permission (self:profile)
```

Comment:
oh I meant like this. If the current comments stay that way even with reformat function, it's fine.
/*
  comments
*/

Reply:
@kangawoo44 I tried that, but unfortunately those does not work in bxb files :( 

![image](https://user-images.githubusercontent.com/102233419/232880968-a4ca3b18-dc1c-4dc0-9761-ee52b5496989.png)

Reply:
oh I forgot these were bxb files 😂